packages/messaging-system/src/drivers/http/use-cases/create-chatroom.js

```diff
@@ -0,0 +1,10 @@
+// Internal dependencies
+const { axios, endpoints } = require('../utils/constants');
+
+const createChatroom = (data) => {
+  axios.post(endpoints.CHATROOMS, data)
+  .then(console.log('Chatroom created successfully '))
```

Comment:
This should refactorized to use async/await and return the chat room information once is created